### ISSUE TITLE
Warning: dependency-cli is not a sfdx command.

### ISSUE THREAD

**Issue by rsoesemann** [forcedotcom/dependencies-cli#10]
I know you will close this as a duplicate but after trying everything in previous issues I am still unable to install this.

**Comment by hkache** [comment]
Hi @rsoesemann - we just came across an issue with sfdx running on node v12.x. 

The solution here was to update node to v13 first and re-install sfdx afterwards. 

To do that follow the [uninstall instructions](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_uninstall.htm) followed by the [reinstall instructions](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli). 

Finally check your `sfdx --version` and make sure it is running on node v13, like `sfdx-cli/7.50.0 darwin-x64 node-v13.11.0`

Did that resolve it for you?

**Comment by RCMoonpie** [comment]
Same here, 

## tl;dr
Windows 10
Installed *sfdx* using: `npm install sfdx-cli --global`
*sfdx* version: `sfdx-cli/7.53.0 win32-x64 node-v13.12.0`
Installed *dependencies-cli* using: `npm install dependencies-cli -global`

## Details
- Had *sfdx* already installed via Windows installer.
- Wanted *dependencies-cli*
- Tried installing by `sfdx plugins:install dependencies-cli`
 Install failed (don't remember details)
- Came here and glanced at the readme.
- Installed NodeJS LTS (12.16.1) via windows installer (including `npm`)
- Tried installing *dependencies-cli* using `npm install dependencies-cli --global`
 Install failed by hanging on step 2 of 4 (I just read of this on a closed issue)
- Came here and glanced at this open issue.
- Uninstalled *sfdx* and NodeJS via Windows Add & Remove Programs.
- Installed NodeJS Current (13.12.0) via Windows installer.
- Installed *sfdx* using `npm install sfdx-cli --global`
- Installed *dependencies-cli* using `npm install dependencies-cli --global`

Here is the output from the install:
```
npm WARN deprecated babel-preset-latest@6.24.1: We're super �  excited that you're trying to use ES2017+ syntax, but instead of making more yearly presets � , Babel now has a better preset that we recommend you use instead: npm install babel-preset-env --save-dev. preset-env without options will compile ES2015+ down to ES5 just like using all the presets together and thus is more future proof. It also allows you to target specific browsers so that Babel can do less work and you can ship native ES2015+ to user � ! We are also in the process of releasing v7, so please give http://babeljs.io/blog/2017/09/12/planning-for-7.0 a read and help test it out in beta! Thanks so much for using Babel �, please give us a follow on Twitter @babeljs for news on Babel, join slack.babeljs.io for discussion/development and help support the project at opencollective.com/babel

npm WARN deprecated crypto@1.0.1: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.

npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)

npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

npm WARN deprecated babel-preset-es2017@6.24.1: �  Thanks for using Babel: we recommend using babel-preset-env now: please read https://babeljs.io/env to update!

npm WARN deprecated babel-preset-es2015@6.24.1: �  Thanks for using Babel: we recommend using babel-preset-env now: please read https://babeljs.io/env to update!

npm WARN deprecated babel-preset-es2016@6.24.1: �  Thanks for using Babel: we recommend using babel-preset-env now: please read https://babeljs.io/env to update!

npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.

npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated

npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

npm WARN deprecated request@2.88.0: request has been deprecated, see https://github.com/request/request/issues/3142

C:\Users\myUserName\AppData\Roaming\npm\componentizer -> C:\Users\myUserName\AppData\Roaming\npm\node_modules\dependencies-cli\bin\run

> dtrace-provider@0.6.0 install C:\Users\myUserName\AppData\Roaming\npm\node_modules\dependencies-cli\node_modules\dtrace-provider
> node scripts/install.js

> core-js@2.6.11 postinstall C:\Users\myUserName\AppData\Roaming\npm\node_modules\dependencies-cli\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

> ejs@2.7.4 postinstall C:\Users\myUserName\AppData\Roaming\npm\node_modules\dependencies-cli\node_modules\ejs
> node ./postinstall.js

Thank you for installing EJS: built with the Jake JavaScript build tool (https://jakejs.com/)

+ dependencies-cli@2.0.1
added 765 packages from 939 contributors in 60.638s`
```

Then this:
`> sfdx dependency`
Gave me this:
` »   Warning: dependency is not a sfdx command.`

**Comment by deniskrizanovic** [comment]
Still have this issue. Followed uninstall instructions.. then updated Node to 13.12.0 .. reinstall everything as per install instructions... still no work..

**Comment by kaulonline** [comment]
Please use the below force installer and hope that should resolve the issue : 
sfdx plugins:install dependencies-cli -f -v

**Comment by aesanch2** [comment]
I encountered a similar issue to others on here on macOS 10.15. I was able to install the plugin only after installing sfdx via npm and not with the the client specific packages.

The client packages are bundled with an older version of node that does not get upgraded when updating the cli.

**Comment by forcetrekker** [comment]
I had this problem and after upgrading node from older version 10.xx to current LTS version `v12.18.3`, the install worked. Like @hkache had mentioned above.
My SFDX and node version now is: `sfdx-cli/7.72.0 win32-x64 node-v12.18.3`

**Comment by mohanads** [comment]
Isn't the proper syntax to install this plugin supposed to be "**sfdx plugins:install dependencies-cli**"? That's the only command that worked for me.
I've followed the uninstall-reinstall instructions and that did not work. Only installing the plugin through the sfdx command worked.
